README.md

```diff
@@ -32,9 +32,7 @@ Deep learning practitioners should use one of the applications
 enabled with oneDNN.
 
 
-In this package oneDNN is built with the Threadpool CPU runtime.
-oneDNN requires the user to implement a Threadpool interface to enable
-the library to perform computations using multiple threads.
+In this package oneDNN is built with the TBB CPU runtime.
```

Comment:
@xhochy, can you please help me understand the reason behind this change? I checked the history of this README.md file and it seems that this line changes every update back and forth.